packages/css/src/__tests__/transform.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('#css-transform', () => {
       const { sheets: actual } = transformCss(`
         margin-left: initial;
         content: 'hello';
-        color: hotpink;
+        color: rebeccapurple;
```

Comment:
Updated colormin didn't convert hotpink to #ff69b4 as they are the same length, a longer color name makes for a better test